tests/DCPS/XTypes/XTypes.mpc

```diff
@@ -1,5 +1,51 @@
-project: dcps_test, dcps_rtps_udp, msvc_bigobj {
+project(XTypes_Publisher): dcps_test, dcps_rtps_udp, msvc_bigobj {
+
+  exename   = xtypes_publisher
+
   TypeSupport_Files {
-    XTypes.idl
+    XTypesPub.idl
+    !XTypesSub.idl
+    !XTypesSubTypeSupport.idl
+  }
+
+  Idl_Files {
+    !XTypesSub.idl
+    !XTypesSubTypeSupport.idl
+  }
```

Comment:
I'm not sure if this section is needed since there is a `TypeSupport_Files` section above.
```suggestion

```